web/src/scripts/webflow/licenseSync.mjs

```diff
@@ -205,8 +205,40 @@ const createNewLicenses = async () => {
   console.info(`Created a total of ${newLicenses.length} licenses`);
 };
 
+const getUnUsedLicenses = async () => {
+  const localLicensesRaw = loadAllLicenses();
+  const localLicensesFiltered = localLicensesRaw.filter((it) => it.webflowId === undefined);
+  const localLicensesIds = new Set(localLicensesFiltered.map((i) => i.webflowId));
+
+  const webflowLicense = await getAllLicensesFromWebflow();
```

Comment:
[dust] Perhaps `webflowLicenseArray` that better describes the objects that are returned from that function.